data-canary/scripts/globalevents/global_server_save.lua

```diff
@@ -10,8 +10,6 @@ local function ServerSave()
 	end
 	-- Updating daily reward next server save
 	UpdateDailyRewardGlobalStorage(DailyReward.storages.lastServerSave, os.time())
-	-- Reset gamestore exp boost count.
-	db.query("UPDATE `player_storage` SET `value` = 0 WHERE `player_storage`.`key` = 51052")
```

Comment:
I think this is necessary to reset the exp boost price no?

Reply:
yes. but only when the server starts up. with this reset query, on every SS (generally 1 per hour), the storage will be reset and the value will be every 30 tc (start price).